packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/TaggedValueEditor.tsx

```diff
@@ -24,19 +24,28 @@ import {
   TimesIcon,
   ArrowCircleRightIcon,
   LongArrowAltUpIcon,
+  VerticalDragHandleIcon,
 } from '@finos/legend-art';
 import { useEditorStore } from '../../EditorStoreProvider.js';
 import {
   type TaggedValue,
   type Tag,
   type Profile,
   isStubbed_PackageableElement,
+  type AnnotatedElement,
 } from '@finos/legend-graph';
 import {
   taggedValue_setValue,
   taggedValue_setTag,
+  annotatedElement_arrangeTaggedValues,
 } from '../../../../stores/graphModifier/DomainGraphModifierHelper.js';
 import type { PackageableElementOption } from '@finos/legend-application';
+import { type DropTargetMonitor, useDrop, useDrag } from 'react-dnd';
+import { getEmptyImage } from 'react-dnd-html5-backend';
+import {
+  type AllTaggedValueDragSource,
```

Comment:
Please do the similar change (as suggested in `StereotypeSelector` and move these to this file